backend/grant/admin/views.py

```diff
@@ -298,16 +302,18 @@ def delete_proposal(id):
     parameter('contributionMatching', type=float, required=False, default=None)
 )
 @admin_auth_required
-def update_proposal(id, contribution_matching):
+def update_proposal(id, contribution_matching, cancel):
```

Comment:
vestigial `cancel` arg?

Reply:
Ah yep, had started off using this endpoint but decided to make it its own. Thanks.